### ISSUE TITLE
Docker install issue

### ISSUE THREAD

**Issue by xingfanxia** [ahmetoner/whisper-asr-webservice#84]
poetry install failed
```
#11 6.118   • Installing ffmpeg-python (0.2.0)
#11 6.120   • Installing h11 (0.14.0)
#11 6.120   • Installing httptools (0.5.0)
#11 6.122   • Installing iniconfig (2.0.0)
#11 6.124   • Installing pluggy (1.0.0)
#11 6.126   • Installing py (1.11.0)
#11 6.128   • Updating more-itertools (9.1.0 -> 9.0.0)
#11 6.129   • Installing python-dotenv (0.21.1)
#11 6.130   • Installing pydantic (1.10.4)
#11 6.135   • Updating setuptools (67.6.0 -> 67.1.0)
#11 6.136   • Installing toml (0.10.2)
#11 6.138   • Installing transformers (4.26.0)
#11 6.143   • Installing torch (1.12.0)
#11 6.145   • Installing starlette (0.20.4)
#11 6.151   • Installing uvloop (0.17.0)
#11 6.155   • Installing websockets (10.4)
#11 6.157   • Installing watchfiles (0.18.1)
#11 6.245 Connection pool is full, discarding connection: pypi.org. Connection pool size: 10
 
#11 6.259 Connection pool is full, discarding connection: pypi.org. Connection pool size: 10
#11 6.270 Connection pool is full, discarding connection: pypi.org. Connection pool size: 10
#11 6.276 Connection pool is full, discarding connection: pypi.org. Connection pool size: 10
#11 6.296 Connection pool is full, discarding connection: pypi.org. Connection pool size: 10
#11 6.338 Connection pool is full, discarding connection: pypi.org. Connection pool size: 10
 
#11 26.60
#11 26.60   _WheelFileValidationError
#11 26.60
#11 26.60   ["In /root/.cache/pypoetry/artifacts/7d/3e/e5/7917acdd005be9659419d0aaf31f173ded60517f74f643685cbb825ba8/torch-1.12.0-cp310-cp310-manylinux1_x86_64.whl, hash / size of torch-1.12.0.dist-info/METADATA didn't match RECORD"]
#11 26.60
#11 26.60   at .venv/lib/python3.10/site-packages/installer/sources.py:289 in validate_record
#11 26.62       285│                         f"In {self._zipfile.filename}, hash / size of {item.filename} didn't match RECORD"
#11 26.62       286│                     )
#11 26.62       287│
#11 26.62       288│         if issues:
#11 26.62     → 289│             raise _WheelFileValidationError(issues)
#11 26.62       290│
 
#11 26.62       291│     def get_contents(self) -> Iterator[WheelContentElement]:
#11 26.62       292│         """Sequential access to all contents of the wheel (including dist-info files).
#11 26.62       293│
#11 26.62
 
#11 ERROR: executor failed running [/bin/sh -c poetry install]: exit code: 1
-----
> [7/7] RUN poetry install:
-----
 
executor failed running [/bin/sh -c poetry install]: exit code: 1
```

**Comment by pjo256** [comment]
Hey! Yeah, this is related to https://github.com/ahmetoner/whisper-asr-webservice/issues/83. An upstream update in `poetry` changed how packages are validated.

For a temporary workaround, you can pin `poetry` in https://github.com/ahmetoner/whisper-asr-webservice/blob/main/Dockerfile#L13 and https://github.com/ahmetoner/whisper-asr-webservice/blob/main/Dockerfile.gpu#L21 - `pip install poetry==1.4.0`

**Comment by ahmetoner** [comment]
closed by #82
